_extras/prereqs.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: Prerequisites
+permalink: /prereqs/
+---
+
+This lesson assumes you have a working knowledge of Python and some previous exposure to the Bash shell. 
```

Comment:
so you guys assume people will not use Windows? I would phrase it like "...and some previous exposure to the terminal.